src/mdrv2.cpp

```diff
@@ -3,14 +3,15 @@
 #include "pcie.hpp"
 
 #include <boost/asio/io_context.hpp>
-#include <phosphor-logging/elog-errors.hpp>
-#include <phosphor-logging/elog.hpp>
```

Comment:
Can you please explain why do we drop elog.hpp and add log.hpp?

Reply:
Reverted.

Reply:
Reverted